backend/sync.js

```diff
@@ -440,7 +440,7 @@ function receiveSyncMessage(backend, oldSyncState, binaryMessage) {
   // changes without applying them. The set of changes may also be incomplete if the sender decided
   // to break a large set of changes into chunks.
   if (message.changes.length > 0) {
-    ;[backend, patch] = Backend.applyChanges(backend, message.changes)
+    [backend, patch] = Backend.applyChanges(backend, message.changes)
```

Comment:
I elected to remove leading `;` from lines where it wasn't necessary, because eslint will detect when it is. These are the only semicolons we use throughout the codebase (at least that I found) but we use this idiom fairly often.